.github/workflows/lint.yaml

```
@@ -0,0 +1,33 @@
+name: "lint"
+
+on:
+  pull_request:
+    types: ["opened", "reopened"]
+  push:
+  schedule:
+    # Run at midnight UTC every day with 15 minutes delay added to avoid high load periods
+    - cron: "15 0 * * *"
+  workflow_dispatch:
+
```

```suggestion

permissions:
  # So the workflow can cancel in-progress jobs
  actions: "write"
  # So `eslint-suggestion-action` can compare commits
  contents: "read"
  # So `eslint-suggestion-action` can post a review
  pull-requests: "write"

```

Just something I've learned we should start doing for better security.